bonus/object.js

```diff
@@ -39,11 +39,11 @@ console.log(allEntries);
  * ==================================
  * When you would use it:
  ***********************************************/
-// CODE HERE
+ console.log(Object.keys(tst));
 
 /**********************************************
  * object.values()
  * ==================================
  * When you would use it:
  ***********************************************/
-// CODE HERE
+ console.log(Object.values(tst));
```

Comment:
Nicely done, but would be great if you could also include instances of when you would use it.